corehq/apps/tzmigration/api.py

```diff
@@ -66,6 +73,19 @@ def phone_timezones_should_be_processed():
         MigrationStatus.IN_PROGRESS, MigrationStatus.COMPLETE)
 
 
+_thread_local = threading.local()
+_thread_local._force_phone_timezones_should_be_processed = False
+
+
+class force_phone_timezones_should_be_processed():
```

Comment:
oops, no I probably just converted it from being a function and didn't notice. Fixed